cli/src/commands/git.rs

```diff
@@ -126,7 +126,7 @@ pub struct GitCloneArgs {
 /// Push to a Git remote
 ///
 /// By default, pushes any branches pointing to
-/// `remote_branches(remote=<remote>)..@`. Use `--branch` to push specific
+/// `(trunk()..@):: & branches()`. Use `--branch` to push specific
```

Comment:
The `& branches()` seems to be unnecessary because of how `jj git push -r` is defined.